src/libraries/Common/tests/System/Security/Cryptography/X509Certificates/CertificateAuthority.cs

```diff
@@ -794,6 +790,7 @@ private enum CertStatus
             Revoked,
         }
 
+        [OverloadResolutionPriority(-1)]
```

Comment:
This seems... unusual. I don't know why this is here. Maybe add a comment why the C# compiler doesn't pick the right overload by default?

Reply:
It's because the new overload that accepts a key factory differs from this one only by an optional parameter. But I'll see if I can just delete it at this point.

Reply:
Had to make some changes for the networking tests (they specified larger keysizes since they don't make as many), but got rid of the legacy.